src/v2/stack_aggregator.py

```diff
@@ -144,12 +143,7 @@ def get_package_details(component: Dict) -> PackageDataWithVulnerabilities:
     recommended_latest_version = pkg_node.get("latest_non_cve_version", [""])[0]
     licenses = version_node.get("declared_licenses", [])
 
-    latest_version = select_latest_version(
-        pkg.version,
-        pkg_node.get("libio_latest_version", [""])[0],
-        pkg_node.get("latest_version", [""])[0],
-        pkg.name
-    )
+    latest_version = pkg_node.get("latest_version", [pkg.version])[0]
```

Comment:
There seems to be a lot happening in the deleted method. Is it this change correct?

Reply:
`libio_latest_version` is no longer populated, it contains stale value. The rest of the candidates are `latest_version` and `pkg.version`. So the selection logic is simplified to consider only  `latest_version`

Reply:
@yzainee Can you please review this piece of code?

Reply:
we dont rely on libio now. so this is fine

Reply:
but i see one problem here. If the user's version is more recent than what we have in our system, then we should not be suggesting that version which we have in our system

Reply:
That is what all the logic around selecting the latest version had. I think you might need to take a look at this.

Reply:
When we don't have users version(which may latest) in graph, it will become unknown package right? I don't think the control will come over here.

Reply:
what is the version is known but the value populated in the graph is incorrect?

Reply:
Is there a chance for that?, `latest_version` would be recomputed on every new version addition to graph right?

Reply:
If it is populated during ingestion, why to check it again?

Reply:
as i said, the original logic was because it wasnt being handled correctly. now if we can make sure that the unknown flow wont come here, i think we should be good to go

Reply:
@yzainee Do you have suggestions to verify this?

Reply:
try with a package which is known but the version unknown and see if this code gets hit or not

Reply:
> try with a package which is known but the version unknown and see if this code gets hit or not

@yzainee , I'm sure this scenario won't hit.

Reply:
then go ahead

Reply:
@prashbnair Any concerns for this?